src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/NamespaceSymbol.cs

```diff
@@ -31,5 +32,18 @@ namespace Microsoft.CSharp.RuntimeBinder.Semantics
 
     internal sealed class NamespaceSymbol : NamespaceOrAggregateSymbol
     {
+        /// <summary>The "root" (unnamed) namespace.</summary>
+        public static readonly NamespaceSymbol Root = GetRootNamespaceSymbol();
+
```

Comment:
Should we consider making this lazy?

Reply:
I don't think there's anything to gain from that. It gets hit (repeatedly) by every single dynamic operation, so there's no chance of it ever not being needed, or even of it not being needed for a while. Indeed it gets hit by stuff that's preloaded eagerly, so making it lazy is counter to that. The cost of null-checks on lazy fields is admittedly small, but if there's nothing to gain for it, it's not worth paying.